shell.nix

```diff
@@ -1,11 +1,10 @@
 { dappPkgs ? (
     import (fetchTarball "https://github.com/makerdao/makerpkgs/tarball/master") {}
-  ).dappPkgsVersions.master-20220325
+  ).dappPkgsVersions.master-20220524
 }: with dappPkgs;
 
 mkShell {
-  DAPP_SOLC = solc-static-versions.solc_0_8_13 + "/bin/solc-0.8.13";
-  SOLC_FLAGS = "--optimize --optimize-runs=200";
```

Comment:
I've removed this line as it is not being used, you can find below that the optimization env variables are being exported so it is still compiling optimized with 200 runs.